### PR TITLE
Merge changes from 19.5 release finalization

### PR DESCRIPTION
**Pull request by mokagio**
Notice that:

1) As usual when a Gutenberg hotfix (https://github.com/wordpress-mobile/WordPress-iOS/pull/18242) lands in the release branch while the next stable release lands on `trunk`, we had conflicts on `Podfile` and `Podfile.lock`. I addressed them by keeping the changes from `trunk`, via `git checkout --theirs` because those represent the latest Gutenberg build. Afterwards, I run `bundle exec pod install` to ensure the `Podfile.lock` consistency.

2) I had to tweak some of the names and subtitles because they didn't comply with the 30 characters limit in App Store Connect. My changes consisted in reverting the files to previous version that passed the ASC test and had already been approved by the translators (or they wouldn't have been in the repo).

### CODE REVIEW

**Review by mokagio — COMMENTED** [review]
- [x] Fastlane  localized `release_notes.txt` updates
- [x] Localized `.strings` updates
- [x] Build number bump in version `.xcconfig` file
- [x] Plus diffs from the PRs that landed on the release branch since the last beta – Or, in this case, and empty diff for `Podfile` and `Podfile.lock` which shows the merge resolution kept the version from `trunk`

File: fastlane/metadata/ru/name.txt [hunk 1]
```diff
@@ -1 +1 @@
-WordPress – средство создания веб-сайтов
\ No newline at end of file
+WordPress
```
Comment (AliSoftware) [inline fastlane/metadata/ru/name.txt:1]: See pxLjZ-6Xb-p2

File: fastlane/metadata/tr/name.txt [hunk 2]
```diff
@@ -1 +1 @@
-WordPress – Web Sitesi Oluşturucu
\ No newline at end of file
+WordPress
```
Comment (AliSoftware) [inline fastlane/metadata/tr/name.txt:1]: See pxLjZ-6Xb-p2

**Review by AliSoftware — APPROVED** [review]
- [x] Changes from bugfix PRs which landed in the release branch
    - Only change since last beta is the Gutenberg 1.72.1 fix, for which you solved the conflict resolution appropriately (hence the bump not being visible in the PR diff) as you explained in your PR description, so 👍
 - [x] Version bump in `.xcconfig` files
 - [x] Latest app and metadata translations pulled from GlotPress
    - No new `.strings` translations which is not surprising since you made a beta recently before that exactly to pull translations in before that final build, so 👌
    - Metadata translations update are in line with new ASO copies + what you noted in pxLjZ-6Xb-p2 for the manual tweaks 👍

**Pull request merged**